app/controllers/openid_connect/token_controller.rb

```diff
@@ -8,9 +8,14 @@ def create
       @token_form = OpenidConnectTokenForm.new(token_params)
 
       result = @token_form.submit
-      analytics.openid_connect_token(**result.to_h)
+      response = @token_form.response
 
-      render json: @token_form.response,
+      analytics_attributes = result.to_h
+      analytics_attributes[:expires_in] = response[:expires_in]
```

Comment:
wait how come we didn't add this to the `extra_analytics_attribute` inside the class?

Reply:
I was hoping to do that, but I couldn't figure out a great way to do it.

It's a bit odd because the extra analytics get called _before_ the response body (which contains the TTL value). It got kind of weird trying to re-do it in a somewhat readable way without unnecessarily validating multiple times or making a call to Redis in an invalid state when we wouldn't want to.